sql/core/src/test/scala/org/apache/spark/sql/DataFrameWriterV2Suite.scala

```diff
@@ -220,6 +220,33 @@ class DataFrameWriterV2Suite extends QueryTest with SharedSparkSession with Befo
       Seq(Row(1L, "a"), Row(2L, "b"), Row(4L, "d"), Row(5L, "e"), Row(6L, "f")))
   }
 
+  test("Overwrite: overwrite by expression: more than one filters") {
```

Comment:
Add JIRA number?

Reply:
Added Jira number and also updated PR description